storage.py

```diff
@@ -56,3 +57,22 @@ async def save_object(
 		)
 		obj._id = result.inserted_id
 		log.debug(f"Inserted {type(obj)} ({obj._id})")
+
+
+async def get_pokemon(
+	id: ObjectId
+) -> Pokemon: # not sure if my type annotations are correct here
+	"""Get pokemon from the database by its ObjectId.
+
+	:param id: The id of the pokemon the user is trying to retrieve
+	"""
+	coll = pokemon(
+	) # I'm not too confident I'm doign this correctly. Should I be doing these inside conditionals like in save_object?
+	result = await coll.find_one(
+		{'_id': ObjectId(id)}
+	) # should I be making the conversion to type ObjectId?
+	if result == None: # if it can't find something, would result == None or undefined?
```

Comment:
undefined is a javascript thing. I don't remember what the result would be off the top of my head.

writing tests can help you figure this stuff out

Reply:
Should tests go in `tests/integration/storage/test_storage.py`? @dyc3

Reply:
yes